app/public/board.js

```diff
@@ -100,6 +100,23 @@ const BOARD = ((rowCount, columnCount) => {
         ROOMS_WITH_THREATS.splice(ROOMS_WITH_THREATS.indexOf(room), 1);
         const roomElement = parseMoveableRoom(room);
         roomElement.setThreat("");
+
+        // If a threat resolves while human is not in room (human enters w/ correct tool then leaves), then set visibility to false
+        // May not be common case once we add minigames to be displayed upon entering room
+        // Note we only once this logic to apply for human player
+        if (HUMAN_STATE.getCurrentRoom() !== room && document.URL.endsWith("human/")) {
+            roomElement.setVisible(false);
+        }
```

Comment:
This seems fair for now. But ultimately, this shouldn't matter, since the human won't be able to leave a room before the threat is completely resolved and removed